draft-yoshino-wish.txt

```diff
@@ -318,26 +329,75 @@ Internet-DrafWiSH: A General Purpose Message Framing over B October 2016
    (see Section 6).  This includes a combination of WiSH and non-WiSH
    media type.
 
-7.2.  Valid UTF-8 Requirement
+
+
+
+
+Yoshino & Zhu          Expires September 25, 2017               [Page 6]
+
+Internet-DraWiSH: A General Purpose Message Framing over Byt  March 2017
+
+
+7.2.  Compression Negotiation
+
+   When layered over HTTP, a client and server MAY choose to negotiate a
+   compression to use by using the standard HTTP "Accept-Encoding" and
+   "Content-Encoding" headers.  A client MAY list offered compression
+   methods as follows:
+
+       Accept-Encoding:
+           web-stream-deflate;
+           client_max_window_bits; server_max_window_bits=10, q=1,
+           web-stream-deflate;
+           client_max_window_bits; server_max_window_bits; q=0.5
+
+   Each element in the header value consists of the identifier of the
+   compression method followed by parameters configuring the method.
+   The "web-stream-deflate" compression method in the example shows how
+   the compression algorithm used for the "permessage-deflate" can be
+   configured for the example.  This example includes the
+   "client_max_window_bits" for WebSocket compatibility which indicates
+   whether or not the client supports the "client_max_window_bits"
+   parameter.  This information is not about the content coding of the
```

Comment:
OK. I'll remove this. I wanted to tell you that I noticed that layering the compression negotiation onto C-E/A-E was not straightforward.